<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchform</class>
 <widget class="QDialog" name="searchform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeSelectComboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>HEX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ASCII</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UTF-16</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
